hologram_dashboard/extension/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { RequestHandler } from '../request-handler.component'
+
+export interface DateAvailability {
+  date: string;
+  isAvailable: boolean;
+}
+
+export interface HologramDataAvailability {
+  sourceType: string;
+  dateAvailabilityList: Array<DateAvailability>
+}
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css'],
+  providers: [ RequestHandler ]
+})
+export class DashboardComponent implements OnInit {
+  dashboard: Array<HologramDataAvailability>;
+  chipper: boolean
+  chipperGDPR: boolean
+  display: boolean
+
+  constructor(private requestHandler: RequestHandler) { 
+    this.chipper = false;
+    this.chipperGDPR = false;
+    this.display = false;
+    this.dashboard = []
+  }
+
+  ngOnInit(): void {
+  }
+
+  renderTemplate(hologram_data_availability: JSON, system: string) {
+    for (let key in hologram_data_availability) {
+      let data_set_availability = {} as HologramDataAvailability;
+      data_set_availability.sourceType = key;
+      data_set_availability.dateAvailabilityList = [];
+      hologram_data_availability[key].forEach(data => {
+        let date_info = {} as DateAvailability;
+        date_info.date = data[0];
+        date_info.isAvailable = data[1];
+        data_set_availability.dateAvailabilityList.push(date_info);
+      });
+      this.dashboard.push(data_set_availability);
+    }
+  }
+
+  onClick(system: string) {
+    if (system === "Chipper") {
+      if (this.chipper) {
+        this.chipper = false;
```

Comment:
nit: add comment that it will hide the chipper dashboard if chipper button is clicked again.

Reply:
can we do something like
if (system == "Chipper") {
  this.chipper = !this.chipper;
} else if (system == "Chipper_GDPR") {
  this.chipperGDPR = !this.chipperGDPR;
}